### PR TITLE
cli(refactor): add version cmd

### PR DESCRIPTION
**Pull request by hi-rustin**
### What problem does this PR solve? 
part of #2168

cc: #2199

split from #2262

### What is changed and how it works?

add test and version cmds.

### Check List 

Tests 

None

Code changes

 - Has exported function/method change

Side effects

None

Related changes

None

### Release note

- No release note

### CODE REVIEW

File: pkg/cmd/cmd.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	"github.com/pingcap/ticdc/pkg/cmd/version"
+	"github.com/spf13/cobra"
+)
+
+// NewCmd creates the root command.
+func NewCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "cdc",
+		Short: "CDC",
+		Long:  `Change Data capture`,
```
Comment (overvenus) [inline pkg/cmd/cmd.go:29]:
```suggestion
		Long:  `Change Data Capture`,
```

File: pkg/cmd/test/test.go [hunk 2]
```diff
@@ -0,0 +1,108 @@
+// Copyright 2021 PingCAP, Inc.
```
Comment (overvenus) [inline pkg/cmd/test/test.go:1]: Seem like we do not need the test cmd any more. Cc @amyangfei

**Review by overvenus — COMMENTED** [review]
Rest LGTM

File: pkg/cmd/test/test.go (hunk 2 above)
Comment (amyangfei) [reply pkg/cmd/test/test.go:1]: Yes, it can be removed

Comment (hi-rustin) [reply pkg/cmd/test/test.go]: Removed! Thanks for your review!

**Review by overvenus — APPROVED** [review]

**Review by amyangfei — APPROVED** [review]

**Comment by hi-rustin** [comment]
💚 💙 💜 💛 ❤️
Thanks!

**Pull request merged**